### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by G-Rath**
Noticed this while working on `no-conditional-expect`, and figured it would be better as a bug release.
Also renamed the messageId to be different from the one used by `no-conditional-expect`.

### CODE REVIEW

File: src/rules/__tests__/no-if.test.ts [hunk 1]
```diff
@@ -552,30 +552,14 @@ ruleTester.run('if statements', rule, {
     },
   ],
   invalid: [
-    {
```
Comment (SimenB) [inline src/rules/__tests__/no-if.test.ts:555]: why is this deleted?

Comment (G-Rath) [reply src/rules/__tests__/no-if.test.ts:555]: Because it's a switch test in the `if` test block; switches are tested in [this group](https://github.com/jest-community/eslint-plugin-jest/pull/587/files#diff-af685d385381b07d87cda847c6b0ae4dR71).

I must have done a bad copy and paste; this test fails because it's data property has it expecting `if` instead of `switch` for `condition`.

Comment (G-Rath) [reply src/rules/__tests__/no-if.test.ts:555]: [Here is it's duplicate](https://github.com/jest-community/eslint-plugin-jest/pull/587/files#diff-af685d385381b07d87cda847c6b0ae4dR179-R193) 🙂

Comment (SimenB) [reply src/rules/__tests__/no-if.test.ts:555]: 👍
